code/workspaces/web-app/src/components/assetRepo/AddRepoMetadataDetails.js

```diff
@@ -60,7 +60,7 @@ export default function AddRepoMetadataDetails() {
       await dispatch(assetRepoActions.addRepoMetadata(metadata));
       notify.success('Metadata set');
     } catch (error) {
-      notify.error(`Error updating metadata: ${error.message}`);
+      notify.error(error.message || error);
```

Comment:
The notify doesn't hang around for very long to be able to read the errors that are returned. What do you think about adding a simple component to render the errors? This could then be put between the form and the submit buttons for example. That way the errors stay on the screen while the user is fixing them and they should be obvious as they are by the button the user has just pressed resulting in the error.

Reply:
I think it would be better to have asyncValidation like some of the other forms - I've added https://jira.ceh.ac.uk/browse/NERCDL-941 "Add async validation to the Add Metadata form, to warn of metadata that already exists" to the backlog.

Reply:
Sounds like a good plan to me